src/Game.js

```diff
@@ -19,18 +19,22 @@ class Game {
   }
 
   generateRound() {
-    if (this.roundTracker <= 2) {
+    // this.roundTracker++;
+
+    if (this.roundTracker < 2) {
       this.currentRound = new Round(this.boards[this.roundTracker], this.players, this.dailyDoubleTurns);
       domUpdates.populateGameBoard(this.currentRound.board)
       this.roundTracker ++;
       this.currentRound.beginTurn();
+      return
     }
-    if (this.roundTracker === 3) {
+    if (this.roundTracker === 2) {
       this.currentRound = new FinalRound(this.boards[this.roundTracker]);
-      domUpdates.populateGameBoard(this.currentRound.board)
+      domUpdates.startFinalRound(this.currentRound.board)
       this.roundTracker ++;
```

Comment:
On lines 34 and 35, it looks like they run for either case of the conditionals. You can bring those lines outside of the condition if they will always run in either case, and then you reduce some repetition.